ciphers/decrypt_caesar_with_chi_squared.py

```diff
@@ -3,10 +3,10 @@
 
 def decrypt_caesar_with_chi_squared(
     ciphertext: str,
-    cipher_alphabet=None,
-    frequencies_dict=None,
+    cipher_alphabet: str = None,
+    frequencies_dict: str = None,
     case_sensetive: bool = False,
-) -> tuple:
+) -> (int, float, str):
```

Comment:
I'm not sure will `Tuple[int, float, str]` work?

Reply:
You can write it both ways, but I'll make the change and commit the relevant changes.